safe-global/safe-wallet-web
# [Multichain] fix: Hide header network selector on non-safe routes [SW-171]

## What it solves

Resolves [SW-171](https://www.notion.so/safe-global/Hide-network-selector-on-non-safe-routes-a3071123877f43b5a56ae630e073298d)

## How this PR fixes it

- Hides the network selector in the header if there is no safe address present

## How to test it

1. Open the welcome page, safe creation, imprint and other non-safe specific pages
2. Observe there is no network selector in the header
3. Open a Safe route
4. Observe there is a network selector in the header

## Screenshots
<img width="758" alt="Screenshot 2024-09-12 at 12 07 09" src="https://github.com/user-attachments/assets/412aad31-d5f6-468c-99a0-ec9739d27d59">
<img width="757" alt="Screenshot 2024-09-12 at 12 07 22" src="https://github.com/user-attachments/assets/dc624ad3-0ea8-4a91-972f-500c8fced3ce">

## Checklist
